the-association/Bylaws.rst

```diff
@@ -55,26 +55,29 @@ Members
 Article 5
 ---------
 
-Any physical person may request to join: if, through their commitment or actions, they have proved their attachment to the aims of the Association, and they are not salaried by the Association.
+Any physical person may request to join: if, through their commitment or actions, they have proved their attachment to the aims of the Association.
 
-The Association comprises: Active members Delegate members Requests to become a member must be addressed to the Board. The Board admits new members and informs the Delegate Assembly.
+The Association comprises: 
+
+* Active members 
+* Delegate members
 
 Membership ceases:
 
 * on death;
 * by written resignation notified to the Board at least 30 days before the end of the financial year;
 * by exclusion ordered by the Board, without reasons duty, with a right of appeal to the Delegate Assembly. Appeals must be lodged within 30 days of the Board’s decision being notified;
-* for non-payment of dues for more than one year
-* for three (3) consecutive absences at a Delegate Assembly (Delegate members only)
+* for non-payment of dues for more than one year;
+* for three (3) consecutive absences at a Delegate Assembly (Delegate members only).
 
 In all cases, the membership fee for the current year remains due. Members who have resigned or who are excluded have no rights to any part of the Association’s assets. Only the assets of the Association may be used to meet commitments. Members have no individual responsibility.
 
 Article 6
 ---------
 
-The Active members include all physical or moral entities elected by the Delegate Assembly who have accepted the present articles and their rules of procedures and who have paid their annual fees.
+The Active members include all entities elected by the Delegate Assembly who have accepted the present articles and their rules of procedures and who have paid their annual fees.
```

Comment:
IMHO, this infringes the Article 5 as it should be physical.

I would have said `The Active members include all members elected by...` as it reused the already defined term (member).

Reply:
+1 do agree on that. I will change this @rousseldenis

Reply:
Changed and committed @rousseldenis